src/components/Register.vue

```diff
@@ -0,0 +1,266 @@
+<template>
+    <title>Login </title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/boxicons@latest/css/boxicons.min.css">
+    <link rel="icon" type="image/x-icon" href="../assets/img/imdb.png">
+    <header>
+      <a href="../html/index.html" class="logo">
+          <i class='bx bxs-movie'></i>IMDB 
+      </a>
+    </header>
+    <main class="loginHeader">
+    <header class=" header_form">
+      <nav class="navBar">
+        <ul>
+          <li><a href="./login.vue" >Sign In</a></li>
+          <li><a href="./Register.vue" class="active">Create Account</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section class="loginForm">
+      <form>
+        <div>
+          <label for="username">Username or Email Address</label>
+          <input type="text" name="username" id="username" placeholder="Enter your username or email address">
+        </div>
+        <div>
+          <label for="password">Password</label>
+          <input type="password" name="password" id="password" placeholder="Enter your password">
+        </div>
+        <div>
+            <label for="password">Password Confrimation</label>
+            <input type="password" name="password" id="password" placeholder="Enter your password">
+          </div>
+        <div class="rememberMe">
+          <input type="checkbox" name="rememberMe" id="rememberMe">
+          <label for="rememberMe">Remember Me</label>
+        </div>
+        <button type="submit">Sign In</button>
+        <p><a href="./login.html">Already have an account ?</a></p>
+      </form>
+    </section>
+   </main>
+</template>
+
+<script setup>
+
+</script>
+
+<style scoped>
```

Comment:
why you didn't use tailwind?

Reply:
The reason I didn't use Tailwind is because I started the project with CSS. However, I will definitely make an effort to learn Tailwind and transition to using it in the future. Thank you for your comment